website/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -483,6 +483,36 @@ components: sources: internal_metrics: {
 			default_namespace: "vector"
 			tags:              _component_tags
 		}
+		buffer_discarded_events_total: {
```

Comment:
I think you'll want to add all of these over here too:

https://github.com/vectordotdev/vector/blob/68965243e33e5dbaaf18c8666f9268fa41f3f08c/website/cue/reference/components/sinks.cue#L629-L634

That is the list of metrics reported by all sinks.

Reply:
Ah I see. I will also add to the `transforms` telemetry since buffers are used there as well. Thanks!